src/components/ButtonPanel.js

```diff
@@ -1,41 +1,49 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import Button from './Button';
 import '../App.css';
 
-function ButtonPanel() {
+const ButtonPanel = (props) => {
+  const { clickHandler } = props;
+
+  const handleClick = (buttonName) => clickHandler(buttonName);
+
   return (
     <div id="button-panel" className="ButtonPanel">
       <div id="group-1">
-        <Button name="AC" color="lightgrey" />
-        <Button name="+/-" color="lightgrey" />
-        <Button name="%" color="lightgrey" />
-        <Button name="÷" />
+        <Button name="AC" color="lightgrey" clickHandler={() => handleClick('AC')} />
```

Comment:
There is no need for passing the values of the buttons here. By setting the value of `clickHandler` here to the `clickHandler` from the button to avoid the error in the browser console. 

```
<Button name="AC" color="lightgrey" clickHandler={clickHandler} />

```
Do this for all the remaining buttons too.